test/e2e_node/conformance/run_test.sh

```diff
@@ -73,9 +73,36 @@ NETWORK_PLUGIN=${NETWORK_PLUGIN:-""}
 # NETWORK_PLUGIN_PATH is the path to network plugin binary.
 NETWORK_PLUGIN_PATH=${NETWORK_PLUGIN_PATH:-""}
 
+# KUBELET_KUBECONFIG_DIR is the path to a dir for the kubelet's kubeconfig file
+KUBELET_KUBECONFIG_DIR=${KUBELET_KUBECONFIG_DIR:-"/var/lib/kubelet"}
```

Comment:
rather than combining an overrideable dir with a magic filename, why not make this `KUBELET_KUBECONFIG=${KUBELET_KUBECONFIG:-"/var/lib/kubelet/kubeconfig"}`

then compute the dir inside `create-kubelet-kubeconfig`
```
local kubeconfig_dir="$(dirname "${1}")"
mkdir -p "${kubeconfig_dir}" ...
```